test/image/mocks/bar_stackrelative_negative.json

```diff
@@ -0,0 +1,13 @@
+{
+    "data":[
+        {"name":"Col1","y":["-1","2","3","4","5"],"x":["1","2","3","4","5"],"type":"bar","uid":"aeb9ea"},
+        {"name":"Col2","y":["2","3","4","-3","2"],"x":["1","2","3","4","5"],"type":"bar","uid":"2f201d"},
```

Comment:
Also, the `uid` should likely be removed from the mock - it's generated per-plot and we should keep mocks as lean as possible.